packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -0,0 +1,37 @@
+{{#showBanner}}
+<div id="banner-brand-message" class="absolute w-full top-0 left-0">
+    <div class="flex justify-center p-2 brand-banner-bg">
+        {{#showPrelaunch}}
+        <div class="flex"/>
+            <div class="flex-none m-logo relative mt-1 mr-4 mb-2 bg-black">
+                <div>
+                    <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABwAAAAQCAYAAAAFzx/vAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAEKSURBVHgB7ZThDYIwEIWLcQBHYAPdQDfQEdxAR9AJdAN0AnECYAOYoGwAG5yv4UiacrX9ofGPL3kEel/v2tI2IaKFUmqnpiqTJGk5voHXsHnv4SdipQ2DSznPkpkGzG2S1YAkaw9vYO2JX6wcBw+jeSBRBTMK68QDeydNwyoFC47q4AdceGKa3wvmtMAd7YILHuVRAB/26Mg/m5Uzgc6JZ7H/MhU4HUqGtqvD5GNspvyqzS4V2t22RurrfHcxBXsVp1guWPAr+hf8uObmgW17UsM9aSvldqMeO/bqybHl45ODqT3MinMNcQpLjz1JvnGM9hbjuyCy3ywpdA5w9lm7w5XA2MvZwjeBqV7FRzhNrCW4kwAAAABJRU5ErkJggg==" alt="{{#t}}mozilla m logo{{/t}}" />
+                </div>
+            </div>
+            <div class="flex-initial max-w-md pr-2">
+                <p class="text-left text-sm font-bold">
+                    {{#t}}Firefox accounts will be renamed Mozilla accounts on Nov 1{{/t}}
+                </p>
+                <p class="text-left text-xs">
+                    {{#t}}You’ll still sign in with the same username and password, and there are no other changes to the products that you use.{{/t}}
+                    <span class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
+                </p>
+            </div>
+        </div>
+        {{/showPrelaunch}}
+        {{#showPostlaunch}}
+        <div className="flex"/>
+            <div>
+                <p class="text-sm font-bold">
+                    {{#t}}We’ve renamed Firefox accounts to Mozilla accounts. You’ll still sign in with the same username and password, and there are no other changes to the products that you use.{{/t}}
+                    <span role="link" class="brand-learn-more cursor-pointer underline">{{#t}}Learn more{{/t}}</a>
+                </p>
+            </div>
+        </div>
+        {{/showPostlaunch}}
+    </div>
+    <div class="absolute w-10 right-0 top-0 pt-3">
```

Comment:
For RTL languages, should this be start-0 to align the close button on the left of the window? https://tailwindcss.com/docs/top-right-bottom-left#using-logical-properties

Reply:
Just FYI - Added link to docs because these logical properties are a fairly new feature of tailwind - we can also use things like `ps` (padding start), `me` (margin end) to support RTL languages :)

Reply:
`start-0`, worked perfectly. Thanks for the tip. Didn't know it was that easy.